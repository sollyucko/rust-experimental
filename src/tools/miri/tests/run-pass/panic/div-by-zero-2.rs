// ignore-windows: Unwind panicking does not currently work on Windows
#![allow(unconditional_panic)]

fn main() {
    let _n = 1 / 0;
}
