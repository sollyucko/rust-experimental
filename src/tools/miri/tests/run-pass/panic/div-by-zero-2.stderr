thread 'main' panicked at 'attempt to divide by zero', $DIR/div-by-zero-2.rs:5:14
